docs/alesco/meeting-minutes/2024-08-21.md

```diff
@@ -0,0 +1,81 @@
+- Cody Robertson
+- Andrew Lukoshko
+- Elkhan Mammadli
+- Ben Thomas
+- Neal Gompa
+  
+### Board Attendees
+- benny Vasquez
+- Alex Iribarren
+
+## Community Attendees
+
+## Decisions Adopted
+1. SPICE support to be added back to AlmaLinux 10
+   - Removed in RHEL 9
+   - Will revisit for enablement in AlmaLinux 9 at a later time
+   - Unanimous decision
+
+## Minutes
+- SPICE Support in AlmaLinux
+  - Neal: Dropped in RHEL due to no RHEL stacks dependent on it.
+    - Cody: https://bugzilla.redhat.com/show_bug.cgi?id=2030592
+  - We can package server-side components in AlmaLinux
+  - We can package client-side components in EPEL
+  - benny: Concerned about further deviation from RHEL
+    - Retracted after clarification of no modifcations to kernel, and only addition of packages
+  - Neal: Virt stack in RHEL remains relatively close to upstream, so good for maintainability of these additions
+  - Neal: Upstream libvirt maintainers are good to work with, can probably propose running their CI against AlmaLinux Kitten 10
+  - Voted to enable/build necessary packages for AlmaLinux 10 unanimously
+  - Tabled discussion of adding back to AlmaLinux 9 at a later time
+- benny: ALESCo Live Community Q&A
+  - No ALESCo members against this
+  - Will be moderated by benny
+  - Potentially quarterly
+  - benny will coordinate a time that works for ALESCo members, maybe Sept. 4 or Sept. 11
+  - Great for transparency and community engagement
+- Disabled HW in RHEL10
+  - Andrew: What should we re-enable
+    - All: Mellanox definitely
+    - Andrew: 10s of devices disabled in EL10 kernel
+    - Jonathan: QLogic & other less-used devices?
+  - Neal: Linux Plumbers will be useful to make connections to RHEL kernel maintainers
+    - They are generally willing to merge things for RHEL-disabled components as long as they don't impact enabled-components.
+    - We probably need a separate process for users to request hardware enablement of devices
+  - Ben: Already has requests within UCL for re-adding support for QLogic cards
+  - Alex (Board): Must be careful when enabling things that might appear easy to flip a flag now but will potentially later become more difficult to maintain.
+  - Jonathan: We can agree on an initial list of things to re-enable that are very common and/or we have direct experience with and vote on the rest as demand arises throughout the AlmaLinux 10 life cycle.
+- Elkhan: Separate kernel for PPC with KVM enabled
+  - Neal: Basically what centos hyperscale does with their kernel - RHEL-ish config (based on Fedora) + enablement on some thingfs
+  - Neal: What if we just re-enable it in the main Alma kernel since it is a flag?
+    - Jonathan: We can mark that feature specifically as experimental, even in the main Alma kernel
+    - Andrew: Just changing the flag isn't enough, a few patches are required to get it to build
+  - Elkhan: Use-case is OpenQA and image building
+    - All: not aware of any use-cases or requests from users
```

Comment:
We may expect OpenStack as one possible use-case/request.
Currently, we are using Power based OpenStack instance provided by [OSU Open Source Lab](https://osuosl.org/).

Feel free to decide whether need to add this part or not. Just shared my thoughts here.